src/main/java/com/seriouscompany/business/java/fizzbuzz/packagenamingpackage/impl/math/arithmetics/IntegerDivider.java

```diff
@@ -41,8 +41,7 @@ public int divide(final int nFirstInteger, final int nSecondInteger) {
 					(double) Constants.INTEGER_ORIGIN_ZERO_VALUE)) {
 				dbRoundedQuotient = Math.floor(dbQuotient);
 			}
-			final int nIntegerQuotient = DoubleToIntConverter.Convert(dbRoundedQuotient);
```

Comment:
these could be useful, can we leave them in?

Reply:
you might need them later?